### ISSUE TITLE
Configurable HTML element

### ISSUE THREAD

**Issue by TomiTakussaari** [dazld/react-on-visible#23]
Thanks for the library, solved one of my problems nicely.

However, currently react-on-visible always renders ```<div>```, but ```<div>``` is not valid child everywhere, eg. under ```<p>```.

Could this element be made configurable ?

```
main.js:12974 Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.
    in div (created by OnVisible)
```

**Comment by dazld** [comment]
thanks for the nice words, I'm glad this helped!

the element which we use to wrap can definitely be made configurable. It sounds like you want to track visibility of an inline element - is that right? something like an inline image or an anchor..? I can have a play with something, and put up a PR.

**Comment by dazld** [comment]
@TomiTakussaari would appreciate if you could take that branch for a spin, and see if it works for you!

**Comment by dazld** [comment]
hm, actually, this doesn't look too controversial to me. I'll publish this as a new version, and then let me know how you get on.

**Comment by dazld** [comment]
published as `1.5.0`

**Comment by TomiTakussaari** [comment]
Yeah, I needed to track inline image, and 1.5.0 seems to do the trick, thanks!
